### ISSUE TITLE
BBS unresponsive after loading saved state

### ISSUE THREAD

**Issue by misterwrong** [pioneerspacesim/pioneer#5066]
### Observed behaviour

Right after loading the game, when opening the BBS, all missions are grayed out and none respond to clicks. When undocking docking again and reopening the BBS, only the missions that shouldn't be available are grayed out and all respond to clicks by opening a form or dialog.

### Expected behaviour

Right after loading the game, when opening the BBS, only the missions that shouldn't be available are grayed out and all respond to clicks by opening a form or dialog.

### Steps to reproduce

1. Start the game 
2. Load a saved game state
3. Open the BBS

__My pioneer version (and OS):__

Version: master (commit ca1ad5566143f5d3b7720a47a8e37f64f3f25e68)
OS: Linux (5.9.12-arch1-1)

**Comment by WKFO** [comment]
Make sure you unpause the game after loading the save file.

**Comment by misterwrong** [comment]
Yes, that is a good work-around, thanks. But why are the missions grayed out when the game is paused?

**Comment by impaktor** [comment]
@misterwrong because the game world is frozen, you've stopped time.

Also stops players from putting the game state in some undefined corner that triggers a crash (search the issue tracker and you'll see plenty of those bugs in the past).

**Comment by nozmajner** [comment]
What's the reason behind loading the game to a paused state?

**Comment by misterwrong** [comment]
@impaktor I can still refuel and request take off while the game is paused. I can also set a route, which makes more sense imo, but it is confusing like this.

**Comment by impaktor** [comment]
Yes, I consider that to be the bug. Only action in paused state should be unpause.

**Comment by The-EG** [comment]
> What's the reason behind loading the game to a paused state?

From what I can tell, you can save a game at pretty much *any* time...I have no idea if it works or if we prevent it, but I've seen some parts of the code that loads the saves that seems to be setup for games that were saved while performing a hyperjump even...

My pure guess is that if you were to save while performing a maneuver or flying close to the surface of a planet with strong gravity, you would want to be able to 'get your bearings' after the save was loaded and not have to react instantly after hitting the load button.

**Comment by osjc** [comment]
Regarding the misbehaving bulletin board, I also noticed that the content completely changes upon load. I suspect each of the starports has its own bulletin board and the relationship between the starports and their boards gets mangled upon save/load.
